auth-web/.eslintrc.js

```diff
@@ -12,9 +12,14 @@ module.exports = {
     'no-console': 'error',
     'no-debugger': process.env.NODE_ENV === 'production' ? 'error' : 'off',
     'sort-imports': 'error',
-    'space-before-function-paren': 1
+    'space-before-function-paren': 1,
+    'no-use-before-define': 'off',
+    '@typescript-eslint/no-use-before-define': ['error', { 'functions': false, 'classes': true, 'variables': true }],
+    'no-unused-expressions': 'off',
+    '@typescript-eslint/no-unused-expressions': ['error', { 'allowShortCircuit': true, 'allowTernary': true }]
```

Comment:
Had to add in these extra rules, apparently we were violating them (they might be new rules?)

Reply:
Better to give a warning (instead of off) for no-use-before-define and no-unused-expressions if there is not much.